### ISSUE TITLE
Unable to deploy rancher-backups when installing using default storage location - S3

### ISSUE THREAD

**Issue by sowmyav27** [rancher/backup-restore-operator#90]
**On 2.5-head commit id: `41f173a ` rancher-backup - `1.0.400-rc3`**

- Create a secret with accessKey and secretKey - AWS creds - from cluster-explorer
- In cluster-explorer, go to apps & market place. Deploy Rancher-backups from charts
- Select Default storage - S3
- Choose credentials created in step 1, enter bucket name, region, endpoint and folder - `04152021`
- Click on create/install
- Error seen:

```
helm upgrade --install=true --namespace=cattle-resources-system --timeout=10m0s --values=/home/shell/helm/values-rancher-backup-1.0.400-rc03.yaml --version=1.0.400-rc03 --wait=true rancher-backup /home/shell/helm/rancher-backup-1.0.400-rc03.tgz
Release "rancher-backup" does not exist. Installing it now.
creating 5 resource(s)
Error: Secret in version "v1" cannot be handled as a Secret: v1.Secret.StringData: ReadString: expects " or n, but found 1, error found in          #10   byte of ...|"folder":1102865,"re|..., bigger context ...|"endpoint":"s3.us-east-2.amazonaws.com","folder":1102865,"region":"us-east-2"},"type":"Opaque"}
|...
```

**Note:**
- Also seen on 2.5.7
- When folder is given in the UI as `"04152021"` with quotes, the upgrade/install of app goes through
- when you enter the same details on a Backup CR, when creating one, no errors seen, backup is seen taken correctly

**Comment by nickgerace** [comment]
Confirmed with @dramich @jiaqiluo @sowmyav27 that this is not a regression. Descoped from `v2.5.8` as a result.

**Comment by thehejik** [comment]
**Test report:**

The issue seems to be fixed on `rancher/rancher:master-head 97ca638` with `Rancher Backups (1.1.100+up1.1.1)` from `dev-v2.6` chart repo `commit: cfd917c824f40d09832595ecddabb4570103aebe`. 

Installation of the chart inc. its CRD passed and when I defined non-existing dir `04152021` (without quotes) in my s3 aws bucket the dir has been created after successful Creation of the backup automatically.

This fix is probably not available yet for `v2.5-head`.

**Comment by thehejik** [comment]
Closing as `Rancher Backups (1.1.100+up1.1.1)` is available only in dev-v2.6
